main.cpp

```diff
@@ -6,9 +6,38 @@
 
```

Comment:
**Reporter**: CPPCHECK
**Rule**: unreadVariable
**Severity**: INFO
**File**: main.cpp L74

Variable &#39;umb_packet_error&#39; is assigned a value that is never used.. Variable &#39;umb_packet_error&#39; is assigned a value that is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1850482568>*